.github/workflows/integration-tests.yml

```diff
@@ -35,22 +35,14 @@ jobs:
           # We need to add each of the lockfile paths so that the global npm cache is populated accordingly
           cache-dependency-path: |
             zip-it-and-ship-it/package-lock.json
-            netlify-plugin-nextjs/package-lock.json
+            test-site/package-lock.json
             netlify-cli/npm-shrinkwrap.json
       - name: Installing netlify-cli
         run: npm ci
         working-directory: netlify-cli
-      - name: Installing netlify-plugin-nextjs
+      - name: Installing test site
         run: npm install
-        working-directory: netlify-plugin-nextjs
-      - name: Patching netlify.toml to change the bundler
```

Comment:
There's no point in doing this anymore, at least for Next.js, since the plugin sets its own bundler anyway.